contrib/completion/bash/docker

```diff
@@ -798,7 +802,7 @@ _docker_run() {
 			__docker_capabilities
 			return
 			;;
-		--cidfile|--env-file)
+		--cidfile|--cgroup-parent|--env-file|--label-file)
```

Comment:
I'm a little confused why we're adding `--cgroup-parent` to be completed via `_filedir`.  The argument there is technically a `_filedir`, but not relative to the current directory or even absolute.  For example:
```console
$ docker run -it --rm --cgroup-parent omg debian
root@4d2c48a73c83:/#
```
And then in another terminal:
```console
$ ls /sys/fs/cgroup/devices/omg/
4d2c48a73c831f03e5ea429fed245e597727736953f44a7063dde7eafc5e6e68
cgroup.clone_children
cgroup.procs
devices.allow
devices.deny
devices.list
notify_on_release
tasks
```